models/models.reviews.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../db/connection.js");
+
+exports.fetchReviews = () => {
+  const reviewQuery = db.query(
+    `SELECT * FROM reviews ORDER BY created_at DESC;`
+  );
+  const commentQuery = db.query(`SELECT * FROM comments;`);
+  return Promise.all([reviewQuery, commentQuery]).then((values) => {
```

Comment:
There is a lot of iteration going on here, I would prefer if this was refactored to use a single request to the database with join rather than making 2 (what could be) very large requests to the db